proto/desmos/posts/v1beta1/polls.proto

```diff
@@ -6,14 +6,14 @@ import "google/protobuf/timestamp.proto";
 
 option go_package = "github.com/desmos-labs/desmos/x/staging/posts/types";
 
-// PollAnswer contains the data of a single poll answer inserted by the creator
-message PollAnswer {
+// Answer contains the data of a single poll answer inserted by the creator
+message Answer {
```

Comment:
@bragaz There is another object named `UserAnswer` to show an answer of a user for a poll. I think using `ProvidedAnswer` may be more explicit. What do you think?

Reply:
I agree with @dadamu that this should be more specific. Another way of achieving this is to move this message inside the `Poll` message:

```protobuf
message Poll {
  ...

  message Answer {
    ...
  }
}
```

Reply:
@dadamu @RiccardoM you're right, I will go with `ProvidedAnswer` then 👍